src/app/model/ChessEngine.ts

```diff
@@ -14,15 +13,15 @@ export class ChessEngine implements ChessModel {
     this.squaresWithPiece = board.squaresWithPiece;
   }
 
-  move(byPlayer: Player, squareFrom: Square, squareTo: Square): (PieceWasMoved | PieceWasCaptured)[] {
+  move(side: Side, squareFrom: Square, squareTo: Square): (PieceWasMoved | PieceWasCaptured)[] {
```

Comment:
Zastanawiam sie czy to side tez nam potrzebne. Czy nie lepiej odczytac ja po prostu z bierki, ktora stoi na squareFrom?

Reply:
Side bym zostawił. Ułatwi to tworzenie wodotrysków w View i SCSS

Reply:
Nie wiem co to ma do modelu :P

Reply:
> Zastanawiam sie czy to side tez nam potrzebne. Czy nie lepiej odczytac ja po prostu z bierki, ktora stoi na squareFrom?

faktycznie...wgl tego nie zauważyłem:
teraz git? - https://github.com/nowakprojects/CodersCamp2020.Project.TypeScript.Chess/pull/78/commits/8499f8441bc7f88ce390b1c7835e864c92673774